docker/bitcoin-regtest/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:buster-slim
```

Comment:
You will be the one who maintains it so keep debian, but you could install blockbook and bitcoin via Nix more easily :stuck_out_tongue:.

Reply:
As our backends are Debian based I wanted to have a similar setup to that.

Reply:
That seems reasonable!